wiki/Troubleshooting-Installation.md

```diff
@@ -79,11 +84,33 @@ Here are some general troubleshooting tips for oppia-android. The specific platf
    
    Try to delete the `.bazelrc` file to solve the above error. 
 
+
 3. **java.lang.ClassNotFoundException: com.android.tools.r8.compatdx.CompatDx**
 
    If, when building the app binary, you encounter a failure that indicates that the CompatDx file cannot be found, this is likely due to you using a newer version of the Android build tools. You can manually downgrade to an older version of build-tools (particularly 29.0.2). Unfortunately, this can't be done through Android Studio but it can be done over a terminal. Follow the instructions listed [here](https://github.com/oppia/oppia-android/issues/3024#issuecomment-884513455) to downgrade your build tools & then try to build the app again.
 
 
+4. If you encounter this error while building bazel in Mac M1:
+      ```
+      ERROR: /Users/OpenSource/oppia-android/model/src/main/proto/BUILD.bazel:167:20: Generating JavaLite proto_library //model/src/main/proto:profile_proto failed: (Segmentation fault): protoc failed: error executing command bazel-out/darwin-opt-exec-2B5CBBC6/bin/external/com_google_protobuf/protoc '--proto_path=bazel-out/android-armeabi-v7a-fastbuild/bin/model/src/main/proto/_virtual_imports/languages_proto' ... (remaining 8 argument(s) skipped)
+
+      Use --sandbox_debug to see verbose messages from the sandbox protoc failed: error executing command bazel-out/darwin-opt-exec-2B5CBBC6/bin/external/com_google_protobuf/protoc '--proto_path=bazel-out/android-armeabi-v7a-fastbuild/bin/model/src/main/proto/_virtual_imports/languages_proto' ... (remaining 8 argument(s) skipped)
+      ```
+
+   **Follow these steps to solve this error:**
+      ```
+      xcode-select --install
+      sudo xcodebuild -licence
+      sudo xcode-select -r 
+      sudo xcode-select -s /Library/Developer/CommandLineTools
+      xcode-select -p 
+      ```
+   After you successfully run above commands, build app using Bazel by following command:
+      ```
+      bazel clean --expunge
+      bazel build //:oppia --noexperimental_check_desugar_deps
+      ```
```

Comment:
Add: the `--noexperimental_check_desugar_deps` flag is explained in the [bazel blog](https://blog.bazel.build/2018/12/19/bazel-0.21.html#android).

Reply:
Done.